WIP new about CircleCI page (#5938)
circleci/circleci-docs: jekyll/_cci2/about-circleci.md

- Server v2.x
---

> Our mission is to empower technology-driven organizations to do their best work. We want to make engineering teams more productive with software builds, tests, and deployments through intelligent automation.

Reviewer: I find this sentence a bit hard to scan but am struggling with an alternative: "We want to make engineering teams more productive with software builds, tests, and deployments through intelligent automation." ... maybe `We make engineering teams more productive. Build, test, and deploy using intelligent automation.` ... or something?

> We work where you work: Linux, macOS, Android, and Windows - on the cloud or on your server. CircleCI provides enterprise-class support and services, with the flexibility of a startup.

Reviewer: ```suggestion
We work where you work: Linux, macOS, Android, and Windows - in the cloud or on your servers. CircleCI provides enterprise-class support and services, with the flexibility of a startup.
```

## What is continuous integration?
{: #what-is-continuous-integration }

**Continuous integration** is a practice that encourages developers to integrate their code into a `main` branch of a shared repository early and often. Instead of building out features in isolation and integrating them at the end of a development cycle, code is integrated with the shared repository by each developer multiple times throughout the day.

Continuous integration is a key step to digital transformation.
 

> Every developer commits daily to a shared mainline. Every commit triggers an automated build and test. If build and test fails, it is repaired quickly - within minutes.

Reviewer: ```suggestion
Every developer commits daily to a shared mainline. Every commit triggers automated tests and builds. If these fail, they are repaired quickly - within minutes.
```

![CircleCI about image]( {{ site.baseurl }}/assets/img/docs/arch.png)

## CircleCI in your workflow
{: #circleci-in-your-workflow}

A software repository on a supported version control system needs to be authorized and added as a project on [circleci.com](https://circleci.com). Every code change then triggers automated tests in a clean container or virtual machine. CircleCI runs each [job]({{site.baseurl}}/2.0/glossary/#job) in a separate [container]({{site.baseurl}}/2.0/glossary/#container) or [virtual machine](https://circleci.com/developer/images?imageType=machine).

CircleCI then sends an email notification of success or failure after the tests complete. CircleCI also includes integrated [Slack and IRC notifications]({{site.baseurl}}/2.0/notifications). Code test coverage results are available from the details page for any project for which a reporting library is added.

CircleCI may be configured to deploy code to various environments, including:
- AWS CodeDeploy
- AWS EC2 Container Service (ECS)
- AWS S3, Google Kubernetes Engine (GKE)
- Microsoft Azure
- Heroku

> Other cloud service deployments are easily scripted using SSH or by installing the API client of the service with your job configuration.

Reviewer: Maybe we should add in that orbs are available for lots of popular platforms to help with deployment configuration too. 

I'm not 100% sure but this line about SSH ans API clients sounds like it might be out of date re how we present deployment now. I'll think about who we can ask.

Also - I know this was in here already so not your language but we try not to use "easy", "easily", "simply" for empathy/not to assume how things feel for readers etc.

Author: @rosieyohannan Yeah good catch -- we can probably add something about deploying with orbs.

## Benefits of CircleCI
{: #benefits-of-circleci }

CircleCI runs nearly one million jobs per day in support of 30,000 organizations. Organizations choose CircleCI because jobs run fast and builds can be optimized for speed.

CircleCI can be configured to run very complex pipelines efficiently with sophisticated [caching,]({{site.baseurl}}/2.0/caching/) [docker layer caching,]({{site.baseurl}}/2.0/docker-layer-caching/) and [resource classes]({{site.baseurl}}/2.0/optimizations/#resource-class) for running on faster machines.

> As a developer using CircleCI you can:

Reviewer: I think full stops at the end of these bullets.

Author: @rosieyohannan I've not been putting full stops on bullet lists after noticing how often we do/don't use them elsewhere on the site. Definitely maybe something maybe for the style guide. If I add them here, I will also add them to all the bullet points that have a whole or partial sentence structure.

Reviewer: Yes! I know we are inconsistent on this (:-() but in my style guide notes I have: use a full stop if the bullet point completes a sentence with the introductory stem. So in this case I think bullet points work.

- Set up [parallelism]({{site.baseurl}}/2.0/parallelism-faster-jobs/) in your [.circleci/config.yml]({{site.baseurl}}/2.0/configuration-reference/) file to run jobs faster
- Configure [caching]({{site.baseurl}}/2.0/caching/) with two simple keys to reuse data from previous jobs in your [workflow]({{site.baseurl}}/2.0/workflows/)
- Configure self-hosted [runners]({{site.baseurl}}/2.0/runner-overview/) for unique platform support
- Access [Arm resources]({{site.baseurl}}/2.0/arm-resources/) for the machine executor
- Use [orbs]({{site.baseurl}}/2.0/using-orbs/), reusable packages of configuration, to integrate with third parties
- Use pre-built Docker [images]({{site.baseurl}}/2.0/circleci-images/) in a variety of languages
- Use the [API](https://circleci.com/docs/api/v2/) to retreive information about jobs and workflows
- Use the [CLI]({{site.baseurl}}/2.0/local-cli/) to access advanced tools locally
- Get flaky test detection with [test insights]({{site.baseurl}}/2.0/insights-tests/)

As an operator or administrator of CircleCI installed on your own servers, CircleCI provides monitoring and insights into your builds and uses [Nomad](https://www.nomadproject.io/) for scheduling.

> See the [CircleCI Operations Guide]({{site.baseurl}}/2.0/server-3-overview/) for complete server documentation.

Reviewer: ```suggestion
See the [CircleCI Operations and Installation Guides]({{site.baseurl}}/2.0/server-3-overview/) for complete server documentation.
```

## Pricing and trial options
{: #pricing-and-trial-options }

Visit CircleCI's [Pricing page](https://circleci.com/pricing/) to view free and paid options. 

> To get started with a free trial, visit the [Sign Up and Try CircleCI]({{site.baseurl}}/2.0/first-steps/) page and get set up with the hosted cloud application.

Reviewer: Rather than linking to this doc I think just link to the signup page, this doc might get repurposed or archived soon I think

Organizations on the free plan are given free credits to use on open source projects. Visit [Building Open Source Projects]({{site.baseurl}}/2.0/oss/) for more information about free containers for public open source projects.

## Additional information
{: #additional-information }

> Any app that runs on Linux, Android, or macOS is supported. Further references:

Reviewer: Either add Windows here or just change this line to introduce core concepts and tutorials?

- [Supported Languages Guide]({{site.baseurl}}/2.0/demo-apps/) document for examples and guides

> - [Core Features]({{site.baseurl}}/2.0/features/) for detailed descriptions and links to procedural documentation

Reviewer: OOo this is such a useful process - I think both this docs are very out of date and in need of reworking/archiving (I'll make notes of these). Instead let's link to: https://circleci.com/docs/2.0/tutorials/ and https://circleci.com/docs/2.0/concepts/ maybe?
